opentracing/example/main.go

```diff
@@ -0,0 +1,130 @@
+package main
+
+// This is a near wholesale copy of the opentracing-go/examples/dapperish.go
+// file. The only thing that has been changed is the addition of an Appdash
+// store, traceapp, and appdash/opentracing Tracer. Everything else is the
+// exact same to show that it changing backends is O(1).
+// One thing I've noticed is the clash of package names, since there is
+// opentracing/opentracing and appdash/opentracing
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"math/rand"
+	"net/http"
+	"os"
+	"runtime"
+	"strings"
+	"time"
+
+	"sourcegraph.com/sourcegraph/appdash"
+	otappdash "sourcegraph.com/sourcegraph/appdash/opentracing"
```

Comment:
I think there is a lot of value in having it be named `opentracing` within the `appdash` repository. Otherwise, users might be confused about the overlap of the `appdash` package and the `appdash/appdashtracer` package if they are not using OT API yet :)

How about `import appdashtracer "sourcegraph.com/sourcegraph/appdash/opentracing"` instead?